content/applications/productivity/iot/config/https_certificate_iot.rst

```diff
@@ -0,0 +1,268 @@
+known as *Secure Sockets Layer* (SSL).
+
+:abbr:`HTTPS (Hypertext Transfer Protocol Secure)` occurs based upon the transmission of
+:abbr:`TLS (Transport Layer Security)`/:abbr:`SSL (Secure Sockets Layer)` certificates, which verify
+that a particular provider is who they say they are.
+
+.. note::
+   In this documentation and Odoo we will use the term "HTTPS certificate"
+   to define the fact that the SSL certificate is valid and allows
+   :abbr:`HTTPS (Hypertext Transfer Protocol Secure)` connection.
+
+
+Why is it needed?
+=================
+
+In order to communicate with certain network devices (in particular payment terminals), the usage of
+:abbr:`HTTPS (Hypertext Transfer Protocol Secure)` is mandatory. If the
+:abbr:`HTTPS (Hypertext Transfer Protocol Secure)` certificate isn't valid, some devices won't be
+able to interact with the :abbr:`IoT (Internet of Things)` Box.
+
+
+How to have obtain a Hypertext Transfer Protocol Secure (HTTPS) certificate
+===========================================================================
+
+The generation of the :abbr:`HTTPS (Hypertext Transfer Protocol Secure)` certificate is automatic.
+
+The :abbr:`IoT (Internet of Things)` Box will send a specific request to `<https://www.odoo.com>`__
+that will send back the :abbr:`HTTPS (Hypertext Transfer Protocol Secure)` certificate if the
+:abbr:`IoT (Internet of Things)` box and database are eligible.
+
+Internet of Things (IoT) eligibility
+------------------------------------
+
+ #. The database should be a **production** instance.
+    The database instance should not be a copy, a duplicate, a staging or a development environment.
+ #. The Odoo subscription must:
+
+    - Have an :guilabel:`IoT Box Subscription` line.
+    - Be in progress.
+
+    If your subscription is linked to your `<https://www.odoo.com>`__ portal user you can check
+    both information by yourself on the portal subscription page.
+
+    .. figure:: https_certificate_iot/sub_example_in_progress.png
+        :align: center
+
+        In this case, both subscriptions are considered "in progress" as the
+        :guilabel:`Filter By: In Progress` was used
+
+    If you are not sure for any of the points, contact your Account Manager or Partner on the
+    matter.
+
+Troubleshooting
+===============
+
+If anything goes wrong during the process of the "HTTPS certificate" generation or reception, a
+specific error will code be given on your :abbr:`IoT (Internet of Things)` box home page.
+
+
+ .. tip::
+    Accessing the :abbr:`IoT (Internet of Things)` box homepage will check for the presence of the
+    "HTTPS certificate" and will attempt its generation if missing.
+    As such, if you have an error on your :abbr:`IoT (Internet of Things)` box home page, you can
+    refresh the :abbr:`IoT (Internet of Things)` home page to see if the error disappears.
+
+    .. seealso::
+        - :ref:`iot/https-certificate`
+
+
+Hypertext Transfer Protocol Secure (HTTPS) certificate errors
```

Comment:
I think we should combine this heading with the previous Troubleshooting. The individual errors aren't bolded and it might be tough to differentiate them apart.

Reply:
so you mean changing each `~~~~~~~~~~` of the errors to `---------` ?

Reply:
Yes that is correct and then combine "Troubleshooting" and "Hypertext Transfer Protocol Secure (HTTPS) certificate errors"